doc/2/guides/essentials/export-import-data/index.md

```diff
@@ -0,0 +1,138 @@
+
+```bash
+# Export index "nyc-open-data" to directory "./dump/nyc-open-data"
+$ kourou index:export nyc-open-data ./dump/
+
+ 🚀 Kourou - Export an entire index content (JSONL format)
+ 
+ [ℹ] Connecting to http://localhost:7512 ...
+  Dumping index "nyc-open-data" in nyc-open-data/ ...
+  Dumping yellow-taxi |==================================================== 100% ||| 42532/42532 documents
+  Dumping green-taxi |==================================================== 100% ||| 21782/21782 documents
+ [✔] Omniscient index exported
+
+$ tree dump/
+
+nyc-open-data
+├─ yellow-taxi
+│ ├── documents.jsonl
+│ └ └── mappings.json
+└── green-taxi
+    ├── documents.jsonl
+    └── mappings.json
+```
+
+## Import public data
+
+Public data can be imported with [Kourou, the Kuzzle CLI](https://github.com/kuzzleio/kourou).
+
+To import your data, you can use the commands [index:import](https://github.com/kuzzleio/kourou/#kourou-indeximport-path) or [collection:import](https://github.com/kuzzleio/kourou/#kourou-collectionimport-path).
+
+These commands use the files generated by the export commands to import the collections and their documents into Kuzzle.  
+
+::: info
+It is possible to import only the data with the `--no-mappings` option.
+:::
+
+```bash
+$ kourou index:import ./nyc-open-data       
+ 
+ 🚀 Kourou - Import a previously dumped index
+ 
+ [ℹ] Connecting to http://localhost:7512 ...
+ [✔] Start importing dump from nyc-open-data in same index
+ [ℹ] Successfully imported 42532 documents in "nyc-open-data:yellow-taxi"
+ [✔] Dump directory nyc-open-data/yellow-taxi imported
+ [ℹ] Successfully imported 21782 documents in "nyc-open-data:green-taxi"
+ [✔] Dump directory nyc-open-data/green-taxi imported
+```
+
+## Export internal data
+
+To export all internal data, you can use [Elasticdump](https://www.npmjs.com/package/elasticdump#multielasticdump) and export all internal collections.  
+
+This will export:
+ - data from the internal collections: `roles`, `profiles`, `users`, `api-keys`, `validations` and `config`.
+ - plugin collections
+
+::: warning
+It is necessary to export the plugin collections if you want to keep the user authentication information because they are stored in the internal storage space of each plugin.  
+:::
+
+```bash
+# Export all internal collections
+$ multielasticdump --direction dump --input http://localhost:9200 --output ./dump --match '%'.
+```
+
+::: info
+If you want to export only roles and profiles, you can use the commands `kourou role:export` and `kourou profile:export`.
+:::
+
+## Import internal data
+
+To import internal data, you can use [Elasticdump](https://www.npmjs.com/package/elasticdump#multielasticdump) and the files generated by the export.  
+
+::: warning
+It is advisable to turn off Kuzzle during import because this operation is not atomic and the interactions performed with the API during import could cause an integrity problem.
```

Comment:
```suggestion
It is advisable to turn off Kuzzle during import because this operation is not atomic and the interactions performed with the API during import could cause integrity problems. 
```